src/storage/gc_worker.rs

```diff
@@ -235,20 +235,20 @@ impl<E: Engine> GCRunner<E> {
 
             if gc_info.found_versions >= GC_LOG_FOUND_VERSION_THRESHOLD {
                 info!(
-                    "[region {}] GC found at least {} versions for key {}",
-                    ctx.get_region_id(),
-                    gc_info.found_versions,
-                    k
+                    "GC found plenty versions for a key";
+                    "region" => ctx.get_region_id(),
+                    "versions count" => gc_info.found_versions,
```

Comment:
avoid space in keys, so that the output can be pretty. e.g. `versions` is enough.